resources/less/editor.less

```diff
@@ -277,38 +340,40 @@
   background-color: #e8f3f6;
 }
 
-.editor-form__form-toolbar-header-text {
-  padding: 2px 0 0 15px;
-  font-size: 14px;
-  color: @blue-text-color;
-  margin-left: 20px;
-}
+.editor-form__preview-buttons {
+  display: flex;
+  align-items: center;
+  margin-left: 15px;
 
-.editor-form__form-toolbar-header-text a {
-  margin-left: 7px;
-  display: inline-block;
-  min-width: 36px;
-  background: #4194bf;
-  color: #fff;
-  padding: 1px 7px;
-  border-radius: 3px;
-  font-size: 13px;
-  text-align: center;
-  min-width: 50px;
-}
+  a {
+    margin-left: 7px;
+    display: inline-block;
+    min-width: 36px;
+    background: #4194bf;
+    color: #fff;
+    padding: 1px 7px;
+    border-radius: 3px;
+    font-size: 13px;
+    text-align: center;
+    min-width: 50px;
+  }
 
-.editor-form__form-toolbar-header-text a:hover{
-  background: #00b5ef;
-}
+  a:first-child {
+    margin-left: 0px;
+  }
 
-.editor-form__form-toolbar-header-text a:before {
-  content: '\f1a3';
-  display: inline-block;
-  font: normal normal normal 14px/1 'Material-Design-Iconic-Font';
-  font-size: inherit;
-  text-rendering: auto;
-  -webkit-font-smoothing: antialiased;
-  padding-right: 5px;
+  a:hover{
+    background: #00b5ef;
+  }
+
+  a:before {
+    content: '\f1a3';
```

Comment:
Tällaset on tosi kryptisia, parempi ois markup tyyliin `<i class="zmdi zmdi-open-in-new"></i>`  (joo tää on vanhaa css:ää, mutta silti).